website/src/docs/hotchocolate/v14/migrating/migrate-from-13-to-14.md

```diff
@@ -100,6 +114,19 @@ builder.Services
     .ModifyOptions(o => o.EnsureAllNodesCanBeResolved = false)
 ```
 
+## IDataLoader<TKey, TValue> arguments now need to be marked as service
```

Comment:
Not sure if this was an intentional change on Hot Chocolate's side.
In our case we're injecting as an `IDataLoader` since our DataLoader is marked as internal, since it depends on an internal dependency.

Reply:
We need to rework this section as this can become a problem. In general you should neither manually register DataLoader with ID nor should you use `IDataLoader<T, V>` to inject things. We have completely overhauled how DataLoader hare handled by the DI. I will try to write something up tonight.

Reply:
I've removed this section.